docs/getting-started-guide.org

```diff
@@ -326,9 +326,44 @@ following lines to your config:
 
 #+begin_src emacs-lisp
   (customize-set-variable 'crafted-init-auto-save-customized nil)
-  (customize-set-variable 'crafted-init-auto-save-selected-passages nil)
+  (customize-set-variable 'crafted-init-auto-save-selected-packages nil)
 #+end_src
 
+Whether or not to save these settings to ~custom.el~ is ultimately a matter of
+personal preference. You'll probably won't notice a difference either way.
```

Comment:
```suggestion
personal preference. You probably won't notice a difference either way.
```

Don't need the contraction in this case.

Reply:
Thanks! I overlooked that there are two "wills" in "you'll" and "won't". Fixed it.